src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -85,7 +96,60 @@ public async Task RunApplicationAsync(CancellationToken cancellationToken = defa
         {
             AspireEventSource.Instance.DcpModelCreationStop();
         }
+    }
+
+    private const string DashboardReservedResourceName = "aspire-dashboard";
```

Comment:
Consider creating:

```c#
internal static class KnownResourceNames
{
    public const string DashboardReservedResourceName = "aspire-dashboard";
}
```

We have some `Known*` classes already that follow this pattern and allow reuse across projects.

Reply:
Addressed in #1775